csharp-basics/exercises/Arrays/Arrays/Exercise8/Program.cs

```diff
@@ -0,0 +1,244 @@
+                {
+                    Console.WriteLine("   +---+   ");
+                    Console.WriteLine("   |   |   ");
+                    Console.WriteLine("   O   |   ");
+                    Console.WriteLine(@"  /|\  |   ");
+                    Console.WriteLine(@"  /    |   ");
+                    Console.WriteLine("       |   ");
+                    Console.WriteLine("===========");
+                    break;
+                }
+                case 2:
+                {
+                    Console.WriteLine("   +---+   ");
+                    Console.WriteLine("   |   |   ");
+                    Console.WriteLine("   O   |   ");
+                    Console.WriteLine(@"  /|\  |   ");
+                    Console.WriteLine("       |   ");
+                    Console.WriteLine("       |   ");
+                    Console.WriteLine("===========");
+                    break;
+                }
+                case 3:
+                {
+                    Console.WriteLine("   +---+   ");
+                    Console.WriteLine("   |   |   ");
+                    Console.WriteLine("   O   |   ");
+                    Console.WriteLine(@"  /|   |   ");
+                    Console.WriteLine("       |   ");
+                    Console.WriteLine("       |   ");
+                    Console.WriteLine("===========");
+                    break;
+                }
+                case 4:
+                {
+                    Console.WriteLine("   +---+   ");
+                    Console.WriteLine("   |   |   ");
+                    Console.WriteLine("   O   |   ");
+                    Console.WriteLine("   |   |   ");
+                    Console.WriteLine("       |   ");
+                    Console.WriteLine("       |   ");
+                    Console.WriteLine("===========");
+                    break;
+                }
+                case 5:
+                {
+                    Console.WriteLine("   +---+   ");
+                    Console.WriteLine("   |   |   ");
+                    Console.WriteLine("   O   |   ");
+                    Console.WriteLine("       |   ");
+                    Console.WriteLine("       |   ");
+                    Console.WriteLine("       |   ");
+                    Console.WriteLine("===========");
+                    break;
+                }
+                case 6:
+                {
+                    Console.WriteLine("   +---+   ");
+                    Console.WriteLine("   |   |   ");
+                    Console.WriteLine("       |   ");
+                    Console.WriteLine("       |   ");
+                    Console.WriteLine("       |   ");
+                    Console.WriteLine("       |   ");
+                    Console.WriteLine("===========");
+                    break;
+                }
+                case 7:
+                {
+                    Console.WriteLine("   +---+   ");
+                    Console.WriteLine("       |   ");
```

Comment:
Vairākas rindas ir vienādas. Ir speciāla komanda, lai izprintēu vairākas reizes vienu un to pašu